jdaviz/configs/imviz/tests/test_astrowidgets_api.py

```diff
@@ -323,6 +323,10 @@ def test_mvp_markers(self):
         # TODO: How to check imviz.app.data_collection.links is correct?
         assert len(self.imviz.app.data_collection.links) == 14
 
+        # Just want to make sure nothing crashes. Zooming already testing elsewhere.
+        # https://github.com/spacetelescope/jdaviz/pull/1971
+        self.viewer.zoom_level = 'fit'
```

Comment:
For the record, I did confirm locally that this test fails on `main` without this patch.